assets/tailwind.config.js

```diff
@@ -1,80 +1,103 @@
 // See the Tailwind configuration guide for advanced usage
```

Comment:
Would be a good opportunity to copy some of the config from MbtaMetro that pulls in preline and default colors.

https://github.com/anthonyshull/mbta_metro/blob/main/assets/tailwind.config.js